optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -273,6 +273,8 @@ def render(self, result, job):
             os.makedirs(html_dir)
             html_path = os.path.join(html_dir, 'results.html')
             self._render(result, html_path)
+            html_symlink_path = os.path.join(job.logdir, 'results.html')
+            os.symlink(os.path.join('html', 'results.html'), html_symlink_path)
```

Comment:
What about the other way around, that is, create it in job.logdir, then creating an html/results.html symlink? That seems more straight forward.